app/webpacker/styles/campaign/_videos.scss

```diff
@@ -0,0 +1,37 @@
+.videos {
```

Comment:
As mentioned yesterday, the videos are slightly smaller (but proportionally) than those on prod

eg on `/events/about-get-into-teaching-events`

I don't think this is an issue - as agreed, I'll raise a separate ticket on videos for a later sprint

Reply:
I'll make them a bit bigger for now and I've got another ticket for looking at videos as a whole later on